docs/hooks/hook_civicrm_alterUFFields.md

```diff
@@ -0,0 +1,37 @@
+# hook_civicrm_alterUFFIelds
+
+## Summary
+
+This hook allows for the modification of the avaliable fields that are permissiable for use within a profile.
+
+## Definition
+
+    hook_civicrm_alterUFFIelds(&$fields) {
+
+## Parameters
+
+-   $fields - a multidimential array of Entities and their fields.
+      For each field contained within the array there are at the following properties
+        - name
+        - title
+        - export
+        - import
+        - hasLocationType
+        - field_type - Entity of the field
+        - other field keys as returned by entity.getfields
+
+## Returns
+
+-   null
+
+## Example
+
+```php
+  /**
+   * Implementation of hook_civicrm_alterUFFIelds
+   */
+  function mte_civicrm_alterUFFIelds(&$fields) {
+    // Incldue grant fields in the permissable array
```

Comment:
sp

Reply:
Is that the stray capital I you're pointing to?

Reply:
I could Incldue that too....

Reply:
both should be fixed now